src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(6);
+        String color = new ColorSupplier().getRandomColor();
+        switch (index) {
+            case 0:
+                return new Square(color, getRandomNumber(1, 14));
```

Comment:
It would be better to move the creation of each shape into a separate private method, like "createRandomSquare()"